lib/ansible/config/manager.py

```diff
@@ -688,6 +688,8 @@ def get_deprecated_msg_from_config(dep_docs, include_removal=False):
         if include_removal:
             if 'removed_at_date' in dep_docs:
                 removal = f"Will be removed in a release after {dep_docs['removed_at_date']}\n\t"
+            elif 'removed_from_collection' in dep_docs:
```

Comment:
Depending on context the key can also be `collection_name` instead of `removed_from_collection`. (It's `collection_name` for deprecated module and plugin options; assuming no normalization is done somewhere that converts `collection_name` to `removed_from_collection`.)

Reply:
aren't those diff tags, remove/tombstone is after deprecation?

Reply:
I'm not sure what you mean. Both show up in deprecations inside plugins and modules. meta/runtime.yml uses another format that does not have this info (but then it's already there from the collection where meta/runtime.yml is in anyway).

https://gist.github.com/felixfontein/06995539f9fc7b06150ab3fe5d80c14e